engines/glk/credits.pl

```diff
@@ -0,0 +1,8 @@
+begin_section("ScummGlk");
```

Comment:
By the way, is this intentionally listed as "ScummGlk" ? Elsewhere it seems this is referred to as just "Glk" ?